packages/prometheus-alertmanager/PKGBUILD

```diff
@@ -48,14 +48,18 @@ package() {
     "${pkgdir}/usr/bin/amtool"
 
   install -dm 750 -o210 -g210 "${pkgdir}/var/lib/alertmanager/data"
-  install -dm 750 -o210 -g210 "${pkgdir}/etc/carbonio/${pkgname}/email-templates"
+  install -dm 750 -o210 -g210 "${pkgdir}/etc/carbonio/${pkgname}/templates"
 
   install -Dm 644 -g210 "${srcdir}/alertmanager.yml" \
     "${pkgdir}/etc/carbonio/${pkgname}/alertmanager.yml"
+
+  install -Dm 644 -g210 "${srcdir}/telegram.tmpl" \
```

Comment:
this is totally wrong. you cannot predetermine the group/user number. That's why groups and users leverage systemd sysusers and tmpfiles in the postinstall stage